src/components/shared/Entity/ValidationErrorSummary/index.tsx

```diff
@@ -9,51 +8,57 @@ import useGlobalSearchParams, {
     GlobalSearchParams,
 } from 'hooks/searchParams/useGlobalSearchParams';
 import { FormattedMessage } from 'react-intl';
-import { useFormStateStore_displayValidation } from 'stores/FormState';
 import {
-    useResourceConfig_hydrationErrorsExist,
-    useResourceConfig_resourceConfigErrorsExist,
-} from 'stores/ResourceConfig';
+    useEndpointConfigStore_errorsExist,
+    useEndpointConfig_hydrationErrorsExist,
+} from 'stores/EndpointConfig';
+import { useFormStateStore_displayValidation } from 'stores/FormState';
 import { hasLength } from 'utils/misc-utils';
 
 interface Props {
     errorsExist: boolean;
     ErrorComponent?: any | boolean;
     hideIcon?: boolean;
     headerMessageId?: string;
+    resourceConfigErrorsExist?: {
+        hydration: boolean;
+        form: boolean;
+    };
 }
 
 function ValidationErrorSummary({
     headerMessageId,
     hideIcon,
     ErrorComponent,
     errorsExist,
+    resourceConfigErrorsExist,
 }: Props) {
     const connectorID = useGlobalSearchParams(GlobalSearchParams.CONNECTOR_ID);
 
+    // Endpoint Config Store
+    const endpointConfigHydrationErrorsExist =
+        useEndpointConfig_hydrationErrorsExist();
+
+    const endpointConfigErrorsExist = useEndpointConfigStore_errorsExist();
+
+    // Form State Store
     const displayValidation = useFormStateStore_displayValidation();
 
-    // Resource Config Store
-    const resourceConfigHydrationErrorsExist =
-        useResourceConfig_hydrationErrorsExist();
+    const hydrationErrorsExist =
+        endpointConfigHydrationErrorsExist ||
+        resourceConfigErrorsExist?.hydration;
 
-    const resourceConfigErrorsExist =
-        useResourceConfig_resourceConfigErrorsExist();
+    const formErrorsExist =
+        errorsExist ||
```

Comment:
Given `errorsExist` only corresponds to details form errors, that selector can be moved into this component and this prop removed.